tests/Oryx.Integration.Tests/NodeEndToEndTests.cs

```diff
@@ -253,8 +253,10 @@ public async Task CanBuildAndRun_NodeApp(string nodeVersion)
                 });
         }
 
-        [Fact]
-        public async Task CanBuildAndRun_NodeApp_WithBuildpack()
+        [Theory]
+        [InlineData("oryxdevms/pack-builder")]
+        [InlineData("heroku/buildpacks")]
```

Comment:
What does testing heroku buildpacks mean here? and why are we having a test for it?

Reply:
I want to make sure our `pack` runner image is compatible with other buildpacks, not just the Oryx buildpack that we maintain.

Reply:
Nice!